### PR TITLE
Internationalize strings

### LINKED ISSUE #362: Internationalize strings
I know this would be a big effort but sooner is always better than later.

### PR DESCRIPTION
**Pull request by Cantido**
This will close #362

### CODE REVIEW

**Comment by drusepth** [comment]
Do the views also need to be changed to use these strings also?

**Comment by Cantido** [comment]
Yep. Check out the rails-i18n gem, those files have some structures we
ought to use, like "Create %{model}". I assume the way to use those is the
"correct" way to do some things.

**Comment by Cantido** [comment]
Phew! That's all of the views in `app/views/main` and `app/views/layouts` done. I did leave out [`_future_navbar.html.erb`](https://github.com/drusepth/Indent/blob/start-i18n/app/views/layouts/_future_navbar.html), since I imagine you still need to create models for it, which would allow us to use `Model.model_name.human`.

File: app/views/layouts/_noscript_notice.html.erb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <noscript>
   <div class="navbar" style="padding: 12px; margin-top: -20px; background: #ff2211; border-bottom: 1px solid #ffbbaa; text-align: center; text: black;">
     <div class="container">
-      Please enable Javascript for the site to function correctly. We promise not to do anything bad!
+      <%= t '.paragraph_1' %>
```
Comment (drusepth) [inline app/views/layouts/_noscript_notice.html.erb]: Could probably use a more descriptive name, but this'll do just fine. :P

Comment (Cantido) [inline app/views/layouts/_noscript_notice.html.erb]: Yeah, I can change it. it's obviously ambiguous in the erb file, but in the
localization file it's named en.views.layouts.noscript_notice.paragraph_1

Comment (drusepth) [inline app/views/layouts/_noscript_notice.html.erb]: Oh wait, I see, it's per page in the translations file. That's pretty slick.

Comment (Cantido) [inline app/views/layouts/_noscript_notice.html.erb]: Yeah! It definitely makes it easier to find the translations now that they
are broken up into files and structured that way. I owe it to the Rails
guide on i18n

**Comment by drusepth** [comment]
The code is looking so much better than it did just a couple months ago. Thanks a ton for all of this.

**Comment by Cantido** [comment]
My pleasure! There is still work to be done but the structure is there. I
just got burned out on the model pages lol. At least the main pages are
done.

**Comment by drusepth** [comment]
@Cantido Oh oops, I merged and then saw your comment about `_future_navbar.html.erb`. Is there more work here you wanted to do outside of that? (AKA did I merge too soon?)

**Comment by Cantido** [comment]
I was expecting to completely internationalize the site before merging, and
most of the model views haven't been touched yet. However, I've done all
the other work necessary, like setting the users locale from the http
headers and setting a default locale. All that's left is the slow work of
extracting strings.

**Comment by drusepth** [comment]
@Cantido I went ahead and reverted the merge and restored the `start-i18n` branch. Feel free to open another pull request whenever is convenient and we'll push all the internationalization changes through together. :+1:

**Pull request merged**